wire-ios/Wire-iOS Tests/Settings/Debug Report/SettingsDebugReportViewModelTests.swift

```diff
@@ -0,0 +1,121 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import XCTest
+
+@testable import Wire
+@testable import WireDataModelSupport
+@testable import WireSyncEngineSupport
```

Comment:
For this test Foundation isn't needed. As far as I understand you don't use any internal members or interfaces of WireDataModelSupport or WireSyncEngine so you don't need to mark them as testable. 

```suggestion
import WireDataModelSupport
import WireSyncEngineSupport
import XCTest

@testable import Wire
```

Reply:
done https://github.com/wireapp/wire-ios/pull/1658/commits/8c85d9ddbe3d4f96ae06fb91a9db48865e963819